vllm/engine/llm_engine.py

```diff
@@ -277,6 +277,10 @@ def add_request(
         # Defensive copy of SamplingParams, which are used by the sampler,
         # this doesn't deep-copy LogitsProcessor objects
         sampling_params = sampling_params.clone()
+        # inject the eos token id into the sampling_params to support min_tokens
+        # processing
+        sampling_params.eos_token_id = self.get_tokenizer_for_seq(
+            seq).eos_token_id
```

Comment:
This can now be simplified
```suggestion
        sampling_params.eos_token_id = seq.eos_token_id
```